packages/unmock-node/src/serialize/index.ts

```diff
@@ -32,8 +32,15 @@ class BodySerializer extends readable.Transform {
 
 function extractVars(
   interceptedRequest: http.IncomingMessage,
-): { method: string; host: string; path: string } {
-  const { host: hostWithPort } = interceptedRequest.headers;
+): {
```

Comment:
General note (unrelated to this PR) - it feels weird to me that we sometimes use arrow notation with `const` and sometimes use the `function` declaration.
Should we just be consistent and use one of them?

Reply:
I use whichever reads clearest to me, I don't think there's really any difference. I think if anyone's bothered by it they can change it, but personally I don't think that's a "company policy" kind of thing...

Reply:
There a few differences, but just because they're so similar, I feel like this is almost like a linting/formatting thing. Not a policy, just consistency across the codebase.

Reply:
Ok, for me readability is more important than consistency but I can try and avoid function declarations.